dune-project

```diff
@@ -1,2 +1,5 @@
-(lang dune 1.0)
+(lang dune 3.0)
```

Comment:
this should then update all the opam files with a lower bound of dune in version 3.0, no?

Reply:
Good point; done.